ci-operator/step-registry/ipi/conf/aws/byo-iam-profile/ipi-conf-aws-byo-iam-profile-ref.yaml

```
@@ -10,5 +10,15 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  env:
+  - name: ENABLE_BYO_IAM_PROFILE_ON_CUMPUTE_NODE
```

```suggestion
  - name: ENABLE_BYO_IAM_PROFILE_CUMPUTE
```

For shorter var name